server/Docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+services:
+  web:
```

Comment:
See the **docker** file is running in the port.
and see whether it running `3306 PORT` or not.